src/js/models/search/SearchOperation.js

```diff
@@ -221,6 +203,24 @@ class SearchOperation {
                 this.fundingAgencies, this.awardingAgencies, this.searchContext));
         }
 
+        // Add cfda query
+        if (this.selectedCFDA.length > 0 && this.searchContext === 'award') {
```

Comment:
You don't need `this.searchContext === 'award'` because `uniqueParams` is only called by `SearchOperation`...whose searchContext is always `award` (unless someone elsewhere in the code is doing something super hacky, in which case it _should_ break).